app/controllers/Api.scala

```diff
@@ -81,6 +81,20 @@ class Api (cc: ControllerComponents, prismDataStore: Prism, prismConfiguration:
       }
     }
 
+    def sourceAccounts: Action[AnyContent] = Action.async { implicit request =>
+      ApiResult.noSource {
```

Comment:
You could use `filter` here instead of noSource. The left hand side of the returned Map is the label of the source - see the `sources` endpoint above. That would make it filterable and incorporate the staleness checks.
Having said that, there probably isn't a reason to filter and will never be stale as it comes purely from config so 🤷‍♂️

Reply:
In the end I decided against this as I only want one result per account - rather than one result per account per source. I think this makes sense, anyone wanting full filtering can use the /sources endpoint